### PR TITLE
Aerie 1.13.0 + Integration test with multiple Aerie versions

### PR DESCRIPTION
**Pull request by cartermak**
- Update constraints query and corresponding integration tests
- Adds test matrix to integration test workflow (but removes 1.12.0 since the constraint schema changes aren't backward-compatible)

### CODE REVIEW

**Review by Mythicaeda — APPROVED** [review]
LGTM!

File: .env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-DOCKER_TAG=v1.12.0
+DOCKER_TAG=v1.13.0
```
Comment (parkerabercrombie) [inline .env:1]: Did you mean to make this 1.13 instead of 1.14?

Comment (cartermak) [reply .env:1]: Nope; I was just focusing on 1.13.0 on this branch and have 1.14.0 updates on a separate branch.

Comment (parkerabercrombie) [reply .env:1]: Ah ok. I saw two PR's come in and must have gotten them confused. 👍

**Pull request merged**